third_party/lua/ltm.h

```diff
@@ -1,9 +1,18 @@
 #ifndef ltm_h
 #define ltm_h
+
 #include "third_party/lua/lobject.h"
+#include "third_party/lua/lstate.h"
 #include "third_party/lua/luaconf.h"
 #include "third_party/lua/tms.h"
 
+
+/*
+* WARNING: if you change the order of this enumeration,
+* grep "ORDER TM" and "ORDER OP"
+*/
+// [jart] moved to tms.h
```

Comment:
Please see the big comment below.

Reply:
Tag is 👍, comment above it should go away (since it's in the other file.)

Reply:
If you look at my "after" screenshot, you'll see that it's the presence of the comment above that allows the diff tool to "re-sync" and make the placement and meaning of the tag obvious.

My goal isn't to be a pain in the ass. My goal is to make this whole process easier for the next person. (In case you weren't aware, there's now a release candidate for 5.4.7, so that opportunity could come pretty soon!) You could definitely disagree with my thinking. But there was definitely thinking behind my changes!

Btw, this isn't the only place I did this. If that's the call I should hunt down any other spots and treat them consistently. (In fact, you can see that I used the same technique with the `ceil()` function that I included in the screenshots.)

Reply:
All right, I see... I think in that case, just document alongside the tag comment that the comment above was left for diff sync purposes, and do so wherever else there's something like that.

I don't doubt that you've been thoughtful while doing this; just trying to understand the thought process and raise questions (even if they don't end with question marks) where I have them.

Reply:
After thinking about this, I decided to tackle this by adding a lot of notes to `README.cosmo`; the notes cover many of the topics that have come up in this review. See what you think.